### PR TITLE
Add travis

### LINKED ISSUE #5: Some CI setup which would run bids-validator on the datasets to guarantee they are all ok
ATM I don't know if we could use this as "BIDS-compliant"  ones ;)

### PR DESCRIPTION
**Pull request by jasmainak**
closes #5 

@chrisfilo could you enable travis so we can see if this works?

### CODE REVIEW

**Comment by chrisfilo** [comment]
travis enabled - could you push a new commit to trigger a build?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+node_js:
+  - "6.11.0"
+
+install:
+  - npm install -g https://github.com/INCF/bids-validator.git
```
Comment (chrisfilo) [inline .travis.yml]: Let's fix it to particular version with `@`

**Comment by jasmainak** [comment]
umm ... are you sure you enabled it? I pushed a commit. Nothing happened

**Comment by chrisfilo** [comment]
hmm it's enabled https://travis-ci.org/INCF/bids-validator, but they are experiencing some issues https://www.traviscistatus.com/incidents/kfgyk0ygvt65

**Comment by chrisfilo** [comment]
try now

**Comment by jasmainak** [comment]
Yayy! worked :-)

**Comment by jasmainak** [comment]
Ready for merge :-)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+node_js:
+  - "6.11.0"
+
+install:
+  - npm install -g https://github.com/INCF/bids-validator.git#0.25.7
+
+script:
+  - for i in $(ls -d */); do bids-validator ${i%%/} --ignoreNiftiHeaders; done
```
Comment (chrisfilo) [inline .travis.yml]: I'm not 100% sure, but I'm afraid that exit code of a for loop is the same as the last command in it - this means things failing before last test will be silenced in Travis. See https://stackoverflow.com/questions/14059342/how-to-get-the-exit-status-a-loop-in-bash

**Comment by jasmainak** [comment]
@chrisfilo comments addressed

**Comment by chrisfilo** [comment]
You're da best!

**Pull request merged**